toolkit/tools/imager/imager.go

```diff
@@ -600,6 +605,9 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap,
 		return
 	}
 
+	// Copy image package manifest from 'installroot' chroot to 'setuproot' chroot
+	installChroot.ExtractFile("image_pkg_manifest_temp1.json", "/image_pkg_manifest_temp2.json")
```

Comment:
Please use named constants instead of hard-coded strings, so that there's only one place where the string itself is set.

Reply:
Sure, updated